docs/how-to-guides/.pages

```diff
@@ -2,3 +2,4 @@ nav:
   - index.md
   - advanced-usage-of-colcon.md
   - how-to-integrate-autoware-with-your-vehicle.md
+  - how-to-integrate-autoware-with-diff-drive-vehicle.md
```

Comment:
How about putting `how-to-integrate-autoware-with-diff-drive-vehicle.md` and the `ackermann_to_diff_drive.png` under the `how-to-integrate-autoware-with-diff-drive-vehicle/` directory?

```suggestion
  - how-to-integrate-autoware-with-diff-drive-vehicle
```

Reply:
Something like this?
![Screenshot from 2022-05-27 13-05-50](https://user-images.githubusercontent.com/44218668/170627180-ca3e653f-49fc-47f2-acb0-2b832d197a8e.png)

The above structure will appear like this. In this case, it feels weird since "How to ~" pages are in the same level as ["How-to guide" top page](https://autowarefoundation.github.io/autoware-documentation/main/how-to-guides/).
![Screenshot from 2022-05-27 13-06-31](https://user-images.githubusercontent.com/44218668/170627212-8d5dcff8-9f1d-4056-b9aa-9aff97f5c02e.png)

Do you know how to solve this?

Reply:
Exactly. I think it's easier to maintain.

> Do you know how to solve this?

Mm, what do you mean? Is that related to this change? :thinking:

Reply:
https://github.com/kminoda/autoware-documentation/tree/add-diff-drive-how-to-guides-with-level-modification
This is the modification I did. Would you check this when you have time?

Reply:
Ah, you mean you want to avoid the title being bold, right?
![image](https://user-images.githubusercontent.com/31987104/170723657-8748c159-6eb6-47cd-8131-39c2f58152b6.png)

I have no good idea to resolve it. :disappointed: 
So you don't have to apply this comment. I'm sorry for the confusion. :bow:

Reply:
Yes that's right.
OK, then I will leave it as it is